src/types/quiz.types.ts

```diff
@@ -0,0 +1,51 @@
+import {
+	ADD__ANSWER,
+	CLOSE__MODAL,
+	OPEN__MODAL,
+	SET__PLAYER__NAME,
+	SET__QUESTIONS,
+} from "../constants";
+
+export type Question = {
+	_id: string;
```

Comment:
You can use `id` here as well, you need not worry about _id from MongoDB
Refer to this: [Mongoose Virtuals](https://mongoosejs.com/docs/tutorials/virtuals.html#virtuals-in-json)
While defining schema use virtual to get id directly from mongoose

Reply:
Sure will try this